Lib/test/test_smtplib.py

```diff
@@ -631,7 +631,11 @@ def testLineTooLong(self):
              'Mrs.C@somewhereesle.com':'Ruth C',
             }
 
-sim_auth = ('Mr.A@somewhere.com', 'somepassword')
+sim_auth_latin = ('Mr.A@somewhere.com', 'somepassword')
+sim_auth_nonlatin = ('Ms.D@example.com', 'p\u03B1sswor\u03B4')
+sim_auth_nonvalid = ('Ms.E@example.com', '\ud800\ud800')
+sim_auth_lists = [sim_auth_latin, sim_auth_nonlatin]
```

Comment:
This should be renamed to something like "valid_sim_auths".

Reply:
Did you mean rename `sim_auth_lists` to `valid_sim_auths`?